doc/cloud-init.md

```diff
@@ -296,3 +296,25 @@ config:
       - type: nameserver
         address: 10.10.10.254
 ```
+
+## How to inject SSH keys on instances
+
+It is possible to inject SSH keys into LXD instances using `cloud-init.vendor-data` or `cloud-init.user-data`. For more information on how to do this using `cloud-config`, see [the cloud-init docs for SSH configuration](https://cloudinit.readthedocs.io/en/latest/reference/yaml_examples/ssh.html) for more information.
+
+Here is an example of a profile configuration that defines a key to be injected on the instance. This enables the owner of the private key to SSH into the instance as a user named `user`:
+
+```yaml
+config:
+  cloud-init.vendor-data: |
+    users:
+      - name: user
+        ssh_authorized_keys: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJFDWcYmMrCZdk9JI29bAiHKD90oEUr8tqK5VvoO8Vcj
+```
+
+It is also possible to inject an additional key for the user `user` on an instance that uses this profile in such a way that the key defined on the profile is still applied. This can be done using the configuration key `cloud-init.ssh-keys.<keyName>`.
```

Comment:
into an instance

Reply:
lets re-order this to promote the new simpler way of injecting ssh keys rather than having the older manual way first.